Nataly_3_2.py

```diff
@@ -0,0 +1,52 @@
+"""
+Author: Nataly
+Date: 15.11.2021
+Task:
+Write a program that reflects these changes and lets you practice with
+the concept of lists. Your task is to:
+step 1: create an empty list named beatles;
+step 2: use the append()method to add the following members of the
+band to the list: John Lennon, Paul McCartney, and George Harrison;
+step 3: use the forloop and the append()method to prompt the user to
+addthe following members of the band to the list: Stu Sutcliffe, and Pete
+Best; input()
+step 4: use thedelinstruction to remove Stu Sutcliffeand Pete Bestfrom
+the list;
+step 5: use the insert()method to addRingo Starr to the beginningof the
+list.
+"""
```

Comment:
This style prefere! Try to use it in the next home works

###############################################################
## 
## Author: Nataly
## Date: 15.11.2021
## Task:
## Write a program that reflects these changes and lets you practice with
## the concept of lists. Your task is to:
## step 1: create an empty list named beatles;
## step 2: use the append()method to add the following members of the
## band to the list: John Lennon, Paul McCartney, and George Harrison;
## step 3: use the forloop and the append()method to prompt the user to
## addthe following members of the band to the list: Stu Sutcliffe, and Pete
## Best; input()
## step 4: use thedelinstruction to remove Stu Sutcliffeand Pete Bestfrom
## the list;
## step 5: use the insert()method to addRingo Starr to the beginningof the
## list.
## 
################################################################